packages/qwik-city/runtime/src/types.ts

```diff
@@ -729,12 +729,14 @@ export interface ActionStore<RETURN, INPUT, OPTIONAL extends boolean = true> {
   >;
 }
 
+type Failed = {
+  failed: true;
+};
+
```

Comment:
For reviewers: 

Should this type be exported?

Reply:
I think it is fine not being exported.